textBasedAdventure.js

```diff
@@ -1,5 +1,1149 @@
+    fight();
+  }
+};
+
+const friend = () => {
+  console.clear();
+  let choiceFriend;
+  console.log(
+    `\n--* Because you are so very brave you nervously wave hello. 
+    Mean Mug smiles back and looks so much better as a result.\n
+    ${userName}: "Why are you looking so mean?"
+    MEAN MUG: "Oh I'm not mean at all. I'm just starving!"
+    ${userName}: "I know the feeling. Ok, let's find you something to eat."
+    MEAN MUG: "Oh. you're so kind. There's a cabin not too far."\n
+  --* Do you ride or fly?\n`
+  );
+  while (choiceFriend !== "ride" && choiceFriend !== "fly") {
+    choiceFriend = rls.question("CHOOSE: Ride? or Fly?\n");
+    choiceFriend = choiceFriend.trim().toLowerCase();
+    if (choiceFriend !== "ride" && choiceFriend !== "fly") {
+      console.log("--* What's the problem here?\n");
+    }
+  }
+  if (choiceFriend === "ride") {
+    ride();
+  } else {
+    console.log("\ns--* You hop onto Minx excitedly.\n");
+    go();
+  }
+};
+
+const reunion = () => {
+  console.log(
+    `\n--* Solara! a voice boomed. Lionel! I new I'd find you here.
+    This is our new friend ${userName}. 
+    Come on let's go eat before the Crumb-snatchers get here.\n`
+    );
+  congratulations();
+};
+
+const sugarLoaf = () => {
+  console.clear();
+  console.log(
+    `\n--* Welcome to the Feast of the SugarLoaf Queen!
+    Nesteled under the canopy of trees was the most magical sight yet.
+    All the characters of this world are here dancing and singing. Fireflies light the night sky
+    twinking all about. Little paper lanterns swing from from the trees swaying in the cool night air. \n`
+    );
+  pass = rls.keyInYN("--* Have you looked in all 3 places?\n");
+  if (pass === true) {
+    reunion();
+  } else {
+    console.log( "--* Don't come back here until you've looked in all 3 places!\n");
+    yes();
+  }
+};
+
+const tearDrop = () => {
+  console.clear();
+  let tearDropDance;
+  console.log(`\n--* You get magically whisked away to Teardrop Valley.
+  At first look it seems a dark and glumy sort of place. Just grey and washed out.\n
+  ${userName}: "Why would you guys come here? It's depressing."
+  MAGICAL FRIEND: "That's to keep out people like you."
+  ${userName}: "You know, you could be nice to me.
+  MAGICAL FRIEND: "Why?"\n
+  Just as you were about to come up with a clever reply thunder cracked and lightning
+  flashed across the sky. As the rain began to fall, the sky turned a lunimous shade of blue.
+  When each raindrop fell and touched something it place became full of color.
+  Soon the entire place was transformed.
+  As the flowers opened little sprites came out to sing and dance.
+  ${userName}: "Ohhhh, now I see why."\n`);
+  while (tearDropDance !== true && tearDropDance !== false) {
+    tearDropDance = rls.keyInYN("--* Do you dance with the sprites?\n");
+    if (tearDropDance !== true && tearDropDance !== false) {
+      console.log("--* Boooooooo!\n");
+    }
+  }
+  if (tearDropDance === true) {
```

Comment:
can shorten to if(tearDropDance) because already truthy 
then just use else. 
If it wasn't true or false we wouldn't have made it to these lines of code.